infra-templates/portworx/config.yml

```diff
@@ -1,7 +1,7 @@
 name: Portworx
 description: |
   Volume plugin for Portworx Elastic Data Fabric, providing scale-out persistent storage for containers.
-version: "2.3.4.0-oci"
+version: "2.5.4.0-oci"
```

Comment:
does this make 2.5.4 the default?

We want the default to be 2.5.3 to be in sync with install.portworx.com

Reply:
fixed in follow-up e8025ff